readme.md

```diff
@@ -1,22 +1,24 @@
-# Jumpstart Live (JSL)
-_In person program for [Ada Developers Academy](http://adadevelopersacademy.org/) students who are not yet confident using loops, arrays, and hashes_
+# JumpStart Live (JSL)
+_In person program for [Ada Developers Academy](http://adadevelopersacademy.org/) students who are not yet well-versed using loops, arrays, and hashes_
 
 ## Overview
-Launched in January 2016, to support newly admitted Ada Developers Academy students who are not yet comfortable and confident using loops, arrays, and hashes. This JSL curriculum is a companion to the [Jumpstart curriculum](https://github.com/Ada-Developers-Academy/jump-start).
+JumpStart Live (JSL) was launched in January 2016 to support newly admitted Ada Developers Academy students who are not yet well-versed using loops, arrays, and hashes. This JSL curriculum is a companion to the [JumpStart curriculum](https://github.com/Ada-Developers-Academy/jump-start).
```

Comment:
drop _who are not yet well-versed using loops, arrays, and hashes_ please

Reply:
Done.